src/core/BufferAttribute.js

```diff
@@ -102,6 +102,16 @@ THREE.BufferAttribute.prototype = {
 
 	},
 
+	setRange: function ( index, typedArray ) {
```

Comment:
One last change... It took me a while to understand what `setRange` was for.
Could you implement in `set()` directly?

```javascript
set: function ( value, offset ) {

	offset !== undefined ? offset *= this.itemArray; offset = 0;
	this.array.set( value, offset );

	return this;

},
```

Undecided whether we should multiply by `this.itemSize` in this one.